example/lib/main.dart

```diff
@@ -117,7 +115,23 @@ class _MyAppState extends State<MyApp> {
                     },
                     value: isHTML,
                   ),
-                  imagePath,
+                  ListView.builder(
+                      shrinkWrap: true,
```

Comment:
`shrinkWrap` is quite expensive. I suggest removing this property and wrapping ListView with `Expanded` widget.

Reply:
As you can't wrap ListView in Expanded, as it's height is not defined, I've wrapped it in Container with height calculated based on items count.
I Hope, just for example app, it's fine.